github/orgs/test_two_factor_required.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+from typing import Any, List
+
+import pytest
+
+from github.orgs.validate_compliance import Criteria
+from .retrieve_github_data import get_all_org_data, OrgInfo
+from . import validate_compliance
+
+# Debugging proper credentials can be challenging, so print out the
+# "viewer" ("authenticated user" in v3 parlance)
+viewer_login: str = "<unqueried>"
+
+
+def get_viewer(endpoint):
```

Comment:
Where is this function used?

Reply:
moved to proper file -- intention is to validate credentials during test collection phase